.travis.yml

```diff
@@ -1,16 +1,38 @@
-language: python
 sudo: required
+dist: xenial
+language: python
 
 
 python:
   - '3.6'
 
+matrix:
+  include:
+    - python: '3.7'
+      env: MYPY="y"
+
 install:
     - pip install --upgrade setuptools
     - pip install codecov
     - pip install -r requirements-dev.txt
 
 script:
-    make cov
+  - |
+      if [ "$MYPY" = "y" ]; then
+          make mypy
+      fi;
+  - make cov
+
 after_success:
     codecov
+
+deploy:
```

Comment:
Put this into the separate stage.

Reply:
I do not want to use stages, they are waste of worker time, since worker allocation and vm setup takes order of  magnitude longer than tests execution.